.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -47,4 +47,6 @@ Text from here to the end of the body will be considered for inclusion in the re
 - [ ] Did you mark the proper milestone (Bug fix: `3.4.x`, [small] API extension: `3.5.x`, API modification or big change: `3.6.0`)?
 - [ ] Did you review?
 - [ ] Did you check whether all relevant Contributor checkboxes have been checked?
-- [ ] Did you mark as `Please Merge`?
+- [ ] Did you do one of the following when ready to merge:
+  - [ ] Squash: You/ the contributor `Enable auto-merge (squash)` and clean up the commit message
+  - [ ] Merge: Label with `Please Merge`
```

Comment:
Now that I think about it, this isn't actually right.

For squash, we do **both** Enable `auto-merge (squash)` and label with `Please Merge`. For rebase+merge, the contributor is supposed to manually update it and then the reviewer (or contributor if they have write access) are supposed to merge with merge commit.

Reply:
i edited this please take a look